Pax.sln

```diff
@@ -0,0 +1,59 @@
+﻿

```

Comment:
What's your opinion on changing commits in this PR? I think the comments on diffs would become orphaned?